crates/flake8_to_ruff/src/plugin.rs

```diff
@@ -331,7 +332,7 @@ pub(crate) fn infer_plugins_from_codes(selectors: &HashSet<RuleSelector>) -> Vec
     .filter(|plugin| {
         for selector in selectors {
             if selector
-                .into_iter()
+                .rules(PreviewMode::Disabled)
```

Comment:
Fine to use "disabled" here, since we don't support preview in `flake8-to-ruff`.